go/strrpos_test.go

```diff
@@ -0,0 +1,127 @@
+package pehape_test
+
+import (
+	"errors"
+	"testing"
+
+	PHP "github.com/teknologi-umum/pehape/go"
+)
+
+func TestStrrpos(t *testing.T) {
+	type param struct {
+		str    string
+		find   string
+		offset int
+	}
+	type test struct {
+		param  param
+		expect int
+	}
+
+	t.Run("It should return error if the given offset is invalid", func(t *testing.T) {
+		tests := []test{
+			{
+				param: param{
+					str:    "GoLang",
+					find:   "Go",
+					offset: 6,
+				},
+			},
+			{
+				param: param{
+					str:    "GoLang",
+					find:   "Go",
+					offset: -7,
+				},
+			},
+		}
+		for _, tt := range tests {
```

Comment:
yg in juga masih `tt` ;D

bisa pake `test` ajasih.
kayak gini 
```suggestion
for _, test := range tests {
```

atau kalau mau pake plural/singular
[ini](https://github.com/teknologi-umum/pehape/pull/46/files#diff-b8982e5e9c4cb77525d65b5ba3217143a68e9d3d7d0a4a8b6640f516195a9928R22) jadi `records` 
[ini](https://github.com/teknologi-umum/pehape/pull/46/files#diff-b8982e5e9c4cb77525d65b5ba3217143a68e9d3d7d0a4a8b6640f516195a9928R22) jadi 
```suggestion
for _, record := range records {
```

begitupun dengan yang lain

Reply:
udah dimerge pak wkw